### ISSUE TITLE
Sustainable accessibility calculation

### ISSUE THREAD

**Issue by attesn** [Traficom/lem-model-system#81]
When calculating sustainable accessibility log should be taken only after summing all exps. 

Now:

`logsum(sustainable) = log(mode_exps[mode1]) + log(mode_exps[mode2]) + ..`

Should be:

`logsum(sustainable) = log(mode_exps[mode1] + mode_exps[mode2] + ..)`

https://github.com/Traficom/lem-model-system/blob/b3920c9f19c76123bd65feb3c2c158584843f13a/Scripts/models/logit.py#L468

**Comment by attesn** [comment]
Also sustainable accessibility could be useful in tour generation model parameters, but is currently available only on last iteration round.
